King-Benx/WeConnect-Android-Java
# Service builder

#### What does this PR do?
This PR delivers a Service Builder

#### Description of Task to be completed?
This holds the base url and instance of the retrofit

#### How should this be manually tested?
1. Clone the repository onto your machine and build the app. 
2. Run ./gradlew build
3. Check the services/ServiceBuilder

#### Background context
NONE
